extensions/sql-migration/src/constants/strings.ts

```diff
@@ -1709,3 +1717,82 @@ export function TDE_MIGRATE_STATUS_ERROR(state: string, error: string): string {
+export const DATA_MIGRATION_INFO = localize('sql.migration.data.migration.info', "Select tables to migrate data to your Azure SQL target");
+export const SCHEMA_MIGRATION_HEADER = localize('sql.migration.schema.migration.header', "Schema migration");
+export const DATA_MIGRATION_HEADER = localize('sql.migration.data.migration.header', "Data migration");
+export const SCHEMA_MIGRATION_CHECKBOX_INFO = localize('sql.migration.schema.migration.checkbox.info', "Migrate schema to target");
+export const SCHEMA_ONLY = localize('sql.migration.schema.only', "Schema only");
+export const DATA_ONLY = localize('sql.migration.data.only', "Data only");
+export const SCHEMA_AND_DATA = localize('sql.migration.schema.data', "Schema and data");
+export const BACKUP_AND_RESTORE = localize('sql.migration.backup.restore', "Backup and restore");
+export const SCHEMA_MIGRATION_STATUS = localize('sql.migration.schema.status', "Schema migration status");
+export const OBJECTS_COLLECTED = localize('sql.migration.schema.objects.collection', "Objects collected");
+export const COLLECTION_STARTED = localize('sql.migration.schema.collection.started', "Collection started");
+export const COLLECTION_ENDED = localize('sql.migration.schema.collection.ended', "Collection ended");
+export const SCRIPT_GENERATION = localize('sql.migration.schema.script.generation', "Script generation");
+export const SCRIPTING_STARTED = localize('sql.migration.schema.script.started', "Scripting started");
+export const SCRIPTING_ENDED = localize('sql.migration.schema.script.ended', "Scripting ended");
+export const SCRIPTED_OBJECTS_COUNT = localize('sql.migration.schema.script.count', "Scripted objects count");
+export const SCRIPTING_ERROR_COUNT = localize('sql.migration.schema.script.fail.count', "Scripting error count");
+export const MIGRATION_TYPE = localize('sql.migration.schema.type', "Migration type");
+export const SCRIPT_DEPLOYMENT = localize('sql.migration.schema.script.deployment', "Script deployment");
+export const DEPLOYMENT_STARTED = localize('sql.migration.schema.script.deployment.started', "Deployment started");
+export const DEPLOYMENT_ENDED = localize('sql.migration.schema.script.deployment.ended', "Deployment ended");
+export const DEPLOYMENT_COUNT = localize('sql.migration.schema.script.deployment.count', "Deployment count");
+export const DEPLOYMENT_ERROR_COUNT = localize('sql.migration.schema.script.deployment.error.count', "Deployment error count");
+export const SCHEMA_MIGRATION_ASSESSMENT_WARNING_MESSAGE = localize('sql.migration.schema.assessment.warning.message', "The detected issues shown below might fail the schema migration. Some of them might be entirely unsupported and the others might be partially supported in Azure SQL Database. \nTherefore, please review the assessment results and make sure all of the issues will not fail the schema migration.\nHowever, it is allowed to proceed the schema migration and DMS will migrate the objects as possible as it can.");
+
```

Comment:
Please avoid using product acronyms like "DMS" as it dilutes the value/name of the service... instead it is recommended that we refer to the service by name "Database Migration Service" or "Azure Database Migration Service"